lib/ui/window/pointer_data.h

```diff
@@ -9,6 +9,29 @@
 
 namespace flutter {
 
+// Must match the button constants in events.dart.
+namespace PointerButton {
```

Comment:
Namespaces are not generally used for constants this way (I should have mentioned this in our discussion). If you're not using enum classes, name them with prefixes instead:
`kMouseButtonPrimary`, `kMouseButtonSecondary`, `kStylusContact`, `kStylusPrimary`, etc.

Reply:
Fixed using `enum` and fixed names, thanks.